apps/help-with-fees/help-with-fees-publicapp/demo-image-policy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: demo-help-with-fees-publicapp
+spec:
+  imageRepositoryRef:
+    name: help-with-fees-publicapp
+  filterTags:
+    pattern: '^pr-678-[a-f0-9]+-(?P<ts>[0-9]+'
```

Comment:
@timja I added this section, do I need it or is the image in demo.yml enough? also the "tag" is created by me in GitHub or by something else?

Reply:
tag is created by your pull request on the jenkins build